amundsen_application/static/js/components/TableDetail/TableIssues/tests/index.spec.tsx

```diff
@@ -35,50 +38,56 @@ describe ('TableIssues', ()=> {
       AppConfig.issueTracking.enabled = true;
     }); 
 
-    it('renders nothing if no issues', () => {
-      const { props, wrapper } = setup({ issues: [] });
-      expect(wrapper.html()).toBeFalsy(); 
-    }); 
-
-    it('renders nothing if issueTracking not enabled', () => {
-      AppConfig.issueTracking.enabled = false;
+    it('renders text if no issues', () => {
       const { props, wrapper } = setup({ issues: [] });
-      expect(wrapper.html()).toBeFalsy(); 
+      expect(wrapper.find('.issue-banner').text()).toEqual(NO_DATA_ISSUES_TEXT);
     }); 
 
     it('renders issues if they exist', () => {
       AppConfig.issueTracking.enabled = true;
       const { props, wrapper } = setup({ issues: [{
         issue_key: 'issue_key', 
         title: 'title',
-        url: 'http://url'
+        url: 'http://url', 
+        status: 'Open', 
+        priority_display_name: 'P2', 
+        priority_name: 'major'
       }]}); 
       expect(wrapper.find('.table-issue-link').text()).toEqual('issue_key'); 
       expect(wrapper.find('.issue-title-name').text()).toContain('title');
     }); 
 
-    it('renders no extra notice if no remaining issues', () => {
-      const { props, wrapper } = setup({ issues: [{
-          issue_key: 'issue_key', 
-          title: 'title',
-          url: 'http://url'
-        }],
-        remainingIssues: 0, 
-        remainingIssuesUrl: null
+    it('renders no link to issues if no issues', () => {
+      const { props, wrapper } = setup({ issues: [],
+        total: 0, 
+        allIssuesUrl: null
       });
       expect(wrapper.find('.table-issue-more-issues').length).toEqual(0); 
     }); 
-    it('renders extra notice if remaining issues', () => {
+
+    it('renders link if there are issues', () => {
       const { props, wrapper } = setup({ issues: [{
           issue_key: 'issue_key', 
           title: 'title',
-          url: 'http://url'
+          url: 'http://url',
+          status: 'Open', 
+          priority_display_name: 'P2', 
+          priority_name: 'Major'
         }],
-        remainingIssues: 1, 
-        remainingIssuesUrl: 'url'
+        total: 1, 
+        allIssuesUrl: 'url'
       });
-      expect(wrapper.find('.table-issue-more-issues').text()).toEqual(SEE_ADDITIONAL_ISSUES_TEXT); 
+      expect(wrapper.find('.table-issue-more-issues').text()).toEqual('View all 1 issues'); 
     }); 
+
+    it('renders Report data issue if no issues ', ()=> {
```

Comment:
A bit confused by this test, we render `ReportTableIssue` all the time now, so if we're checking for it that's what we'd want to check.